### DISCUSSION TITLE
Taxonomy count

### DISCUSSION THREAD

**Discussion by sagarbehere** [razonyang/hugo-theme-bootstrap#311] [Q&A] [answered]
Some of the example sites using this theme (for example, [this one](https://www.sudipg.com.np/)) have a small circular badge in the taxonomies sidebar, which is an integer showing how many posts have that tag/category. However, I just installed the theme and it seems this feature is not present. Below is a screenshot of the exampleSite after installing the theme (to the left) and another site that is using the theme (to the right)

![Screenshot from 2021-12-24 00-13-20](https://user-images.githubusercontent.com/1622959/147399180-630b34d3-9a2c-43c0-bbe1-be4afe4c74b2.png)

How to get those circles with the tag count? 

As far as I can make out, this theme had that taxonomy count feature until tag v0.44.0 and it got removed in v0.45.0. In v0.44.0, the `layouts/partials/sidebar/taxonomies.html` had `<span class="badge rounded-pill">{{ $taxonomy.Count }}</span>` but that got removed?. I can confirm that adding that back to the template brings back the tag count.. but it looks ugly. Making it pretty probably requires additional CSS settings that I do not have the expertise for.

Any idea why this got removed? How to bring it back?

**Comment by razonyang** [comment]
Yep, it was removed. I'll bring it back later.

**Comment by razonyang** [comment, marked as answer]
Implemented by #313 
You'll need to upgrade the theme, and set the `countTaxonomyPosts` parameter to `true`(`params.toml`), in order to turn it on.

**Comment by sagarbehere** [comment]
@razonyang Wow, you fixed this amazingly fast. Thank you very much!
